variables.tf

```diff
@@ -72,3 +72,14 @@ variable "threat_intel_list_path" {
   default     = ""
   type        = "string"
 }
+
+
+variable "accept_invite" {
+  description = "Accept invitation from a master account."
+  default     = false
+}
+
+variable "accepter_master_account_id" {
+  description = "Account ID for Guard Duty Master. Required if accept_invite = true"
+  default = ""
```

Comment:
Same `type = "string"` here. And please put all the `=` in a straight line. My OCD again...

Reply:
will do... I thought I'd ran fmt before pushing, but missed this file.